templates/object_preview.html

```diff
@@ -1,10 +1,10 @@
 {% load thumbnail %}
 
-<a class="artist-details name" href="{{ object.url }}">
-    {% thumbnail img "250x250" crop="center" as thumb %}
-        <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ object.name }}" />
-    {% endthumbnail %}
-    <div class="artist-details artist-name">
-        {{ object.name }}
-    </div>
-</a>
+<div data-content="{{ object.name }}" class="name">
```

Comment:
Taking a look at the artist list and music list pages, I find it's actually really hard to tell anything is on the page at all with this new object preview. Is that on purpose?